packages/styled/package.json

```diff
@@ -75,17 +125,24 @@
       "default": "./macro.js"
     }
   },
+  "imports": {
+    "#is-development": {
+      "development": "./src/conditions/true.js",
+      "default": "./src/conditions/false.js"
+    },
+    "#is-browser": {
+      "worker": "./src/conditions/false.js",
+      "browser": "./src/conditions/true.js",
+      "default": "./src/conditions/false.js"
+    }
```

Comment:
this is an important question: do we assume that `default` means `node`? or do we introduce a new `node` entry and make `default` contain an actual runtime check?

Reply:
Yeah, the default bundle should definitely contain the runtime check and like we could have `worker` and `node` point to just false, idk if it's worth it since the server-side bundle size consideration is nowhere near as big as client-side and maybe this wouldn't actually be a problem but you could e.g. use JSDOM in Node for tests in which case you'd want `isBrowser` to be true

Reply:
i kinda feel like probably JSDOM users should use a jsdom condition or something but that's definitely not popular so ye, let's not add a node condition right now

Reply:
Btw. `jest-environment-jsdom` is already preconfigured with the browser condition (in new Jest versions that is). So `isBrowser` would already be `true` in that context